### PR TITLE
feat:Adding more functionality to Navigation Drawer

### LINKED ISSUE #1033: Adding more functionality to Navigation Drawer
### Feature Request

**Feature Description**
- Currently the Navigation Drawer doesn't have the functionality to edit event which has been implemented in the Event Brite app. I would like to add this to the Orga App.
- **Logout** present in the Orga App can be refactored to **Logged in as** as given in the Event Brite app.
- Events can be changed to  **My Events**

**Screenshots**
ezgif com-resize 1

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #1033 

Changes: 
All the below changes were done in accordance with the event brite app.

- Ability to edit events added in the navigation drawer
- **Events** changed to **My Events**
-  **Feedback** option was moved to bottom of the navigation drawer.  

Screenshots for the change: 
whatsapp image 2018-06-08 at 22 56 59 1

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Why is Feedback moved to bottom?

**Comment by sridharjajoo** [comment]
@iamareebjamal Event brite has it at the bottom.
whatsapp image 2018-06-08 at 09 43 04 1 1

**Comment by iamareebjamal** [comment]
Both Feedbacks are fundamentally different. One is the Event Feedback which is tied to the selected Event, and other is App Feedback(in Eventbrite)

**Comment by sridharjajoo** [comment]
Apologies, I probably didn't take that into notice. I'll change its position.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -79,7 +79,7 @@
     <string name="check_in_status">Check In Status</string>
     <string name="dashboard">Dashboard</string>
     <string name="attendees">Attendees</string>
-    <string name="events">Events</string>
+    <string name="events">My Events</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: Use new string

File: app/src/main/java/org/fossasia/openevent/app/core/main/DrawerNavigator.java [hunk 2]
```diff
@@ -31,8 +33,14 @@ void setLogoutDialog(AlertDialog logoutDialog) {
...
 
-        if (id == R.id.nav_logout)
+        if (id == R.id.nav_logout) {
             showLogoutDialog();
+        }
+        else if (id == R.id.nav_edit_event) {
+            Intent intent = new Intent(context, CreateEventActivity.class);
+            intent.putExtra(EVENT_ID, fragmentNavigator.getEventId());
+            context.startActivity(intent);
+        }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/main/DrawerNavigator.java]: Follow Java style guidelines

File: app/src/main/java/org/fossasia/openevent/app/core/main/DrawerNavigator.java [hunk 3]
```diff
@@ -16,6 +17,7 @@
     private final FragmentNavigator fragmentNavigator;
 
     private AlertDialog logoutDialog;
+    public static final String EVENT_ID = "event_id";
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/main/DrawerNavigator.java]: Isn't it already present in CreateEventActivity

**Pull request merged**